astroquery/heasarc/core.py

```diff
@@ -0,0 +1,216 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import print_function
+
+from ..query import BaseQuery
+from ..utils import commons
+from ..utils import prepend_docstr_noreturns
+from ..utils import async_to_sync
+from . import conf
+
+
+# export all the public classes and methods
+__all__ = ['Heasarc', 'HeasarcClass']
+
+
+@async_to_sync
+class HeasarcClass(BaseQuery):
+    URL = conf.server
+    TIMEOUT = conf.timeout
+
+    def query_object_async(self, object_name, mission, cache=True,
+                           get_query_payload=False):
+        """TODO: document this!
+
+        This method is for services that can parse object names. Otherwise
+        use :meth:`astroquery.heasarc.TemplateClass.query_region`.
+        Put a brief description of what the class does here.
+
+        Parameters
+        ----------
+        object_name : str
+            name of the identifier to query.
+        get_query_payload : bool, optional
+            This should default to False. When set to `True` the method
+            should return the HTTP request parameters as a dict.
+        verbose : bool, optional
+           This should default to `False`, when set to `True` it displays
```

Comment:
The indent here is wrong, and I guess you got the sphinx failure because of that.